apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -224,6 +225,16 @@
         .dynamic(true)
         .buildWithDefault(Collections.singletonList(WildcardMatcher.valueOf("(?-i)*Nested*Exception")));
 
+    private final ConfigurationOption<List<WildcardMatcher>> ignoreExceptions = ConfigurationOption
+        .builder(new ListValueConverter<>(new WildcardMatcherValueConverter()), List.class)
+        .key("ignore_exceptions")
+        .tags("added[1.11.0]")
+        .configurationCategory(CORE_CATEGORY)
+        .description("A list of exceptions that should be ignored and not reported as errors.\n" +
```

Comment:
We must document that we allow patterns and not only exact names, including examples. See what we do in `sanitize_field_names` for example, and how it looks like [in the documentation](https://www.elastic.co/guide/en/apm/agent/java/current/config-core.html#config-sanitize-field-names)

Reply:
good point, I'll do that.